sections/02_intro.md

```diff
@@ -1,97 +1,155 @@
+insufficient to support deeper architectures, however new datasets that will
+support deep methods are expected; or 3) as building blocks to be combined with
+other non-neural-network-based approaches at subsequent stages.
 
-In this review, we are interested in the application of deep learning to 
-topics of biomedical importance. This covers a large range of topics, which 
-we divide into three broad classes based on their applied areas. We then 
-briefly introduce the types of questions, approaches and data which are 
-typical for each class in the application of deep learning.
+### Will deep learning transform the study of human disease?
+
+With this review, we specifically set out to address the question: "what would
+need to be true for deep learning to transform how we categorize, study, and
+treat individuals to maintain or restore health?." We set a high bar for the
+term "transform." Specifically we sought to identify whether deep learning was a
+disruptive innovation that would induce a strategic inflection point on the
+practice of biology or medicine. There are numerous examples where deep learning
+has been applied to biological problems and produced somewhat improved results,
+and there are numerous reviews that have focused on general applications of deep
+learning in biology [@doi:10.1038/nbt.3313 @doi:10.1002/minf.201501008
+@doi:10.3109/10409238.2015.1135868 @doi:10.1021/acs.molpharmaceut.5b00982
+@doi:10.15252/msb.20156651 @doi:10.1093/bib/bbw068]. We sought cases where deep
+learning was enabling researchers to solve challenges that were previously
+considered infeasible, or if it made difficult, tedious, and non-routine
+analyses routine.
+
+Based on our guiding question, we focused on the application of deep learning to
+topics of biomedical importance. This covers a large range of topics. We divided
+these into three broad classes based on their applied areas. We briefly
+introduce the types of questions, approaches and data which are typical for each
+class in the application of deep learning.
 
 #### Disease and Patient Categorization
 
-One important topic in the biomedical field is the accurate classification of 
-diseases and disease subtypes. In the oncology field, current "gold standard" 
-approaches are limited to either histological approaches, requiring manual 
-human expertise, or shallow molecular markers, such as the cell surface 
-receptors or small panels of genes. One example is the current PAM50 approach 
-in classifying breast cancer, which utilizes the expression of 50 marker 
-genes in order to divide breast cancer patients into four subtypes. 
-Significant heterogeneity still remains within these four subtypes 
-[@doi:10.1200/JCO.2008.18.1370 @doi:10.1158/1078-0432.CCR-13-0583]. Given the 
-increasing wealth of molecular data available, it seems that a more 
+One important topic in the biomedical field is the accurate classification of
```

Comment:
Are these blurbs included in the intro, or are they placeholders for the larger sections? I don't think they belong in the intro given that your last paragraph read like a concluding paragraph in an intro that setup discussion of these topics.